README.md

```diff
@@ -29,6 +29,7 @@ Move it to ~/.cache/yarn (see https://yarnpkg.com/lang/en/docs/cli/cache/ for in
 ```
 git clone git@github.com:jsdrupal/drupal-admin-ui.git
 cd drupal-admin-ui
+uid=$(id -u) gid=$(id -g) docker-compose buil
```

Comment:
```suggestion
uid=$(id -u) gid=$(id -g) docker-compose build
```

What happens with windows machines here? I could imagine `id` is not available there?

Reply:
@dawehner I'm not sure :thinking: